### ISSUE TITLE
[Bug]: [Auto layout] Unable to resize 2d widgets in mobile viewport , if they were initially placed  on desktop view

### ISSUE THREAD

**Issue by kamakshibhat-appsmith** [appsmithorg/appsmith#20894]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Unable to resize 2d widgets in mobile viewport , if they were initially placed  on desktop view

### Steps To Reproduce

1. DnD a Form on to canvas
2. resize the canvas to mobile viewport using resize indicators on the canvas
3. Try to increase size of form
https://www.loom.com/share/6122e0ab1b5f4d8aabad92ede94cce8c

### Version

DP

**Comment by Sripriya93** [comment]
Waiting on decision if we need vertical resizers or Auto height. QA on hold.
